sessionctx/variable/sysvar.go

```diff
@@ -707,6 +707,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TiDBSlowLogThreshold, strconv.Itoa(logutil.DefaultSlowThreshold)},
 	{ScopeSession, TiDBRecordPlanInSlowLog, strconv.Itoa(logutil.DefaultRecordPlanInSlowLog)},
 	{ScopeSession, TiDBEnableSlowLog, strconv.Itoa(logutil.DefaultTiDBEnableSlowLog)},
+	{ScopeSession, TiDBLogLevel, logutil.DefaultLogLevel},
```

Comment:
Since this viriable affects a global setting, could we only support `ScopeGlobal`?

Reply:
Using `ScopeGlobal` makes the config item visible to all TiDB instances in the same cluster by storing this config item to TiKV and other TiDB instances will load it later.
Actually this config item should be `ScopeInstance` (or `ScopeServer`), but now this scope level is not supported by TiDB, which is a known issue.
Some other config items like `TiDBSlowLogThreshold`, `TiDBRecordPlanInSlowLog` also have this problem...

Reply:
Seems this function is duplicated with `update ... cluster_config`?

Reply:
Is there any document about this feature? `update ... cluster_config`? @lonng

Reply:
And I will discuss with our PM about this config item's scope level.